assets/infrastructure-providers/infrastructure-azure.yaml

```diff
@@ -105,6 +106,9 @@ data:
             - mountPath: /tmp/k8s-webhook-server/serving-certs
               name: cert
               readOnly: true
+            - mountPath: /var/run/secrets/azure/tokens
```

Comment:
The configuration secret made by CCO will set this as `/var/run/secrets/openshift/serviceaccount/token` which is a default used across the board.

Reply:
Would be good to cross reference this with the `AZURE_FEDERATED_TOKEN_FILE` env var.

Reply:
Is it going to get overwritten then? I can't change that as this file is generated by `make assets` pulling from upstream.

Reply:
ACK, we don't have plumbing to override the path in the secret CCO generates for the CredentialsRequest but we could add that if needed.

Reply:
The `CredentialsRequest` recently grew a `.spec.cloudTokenPath` [field](https://github.com/openshift/cloud-credential-operator/blob/de95fe7be355adaa7e0b4a7a66d3226b1936b0f8/pkg/apis/cloudcredential/v1/types_credentialsrequest.go#L74) that we could plumb into ccoctl for this.

Reply:
https://github.com/openshift/cloud-credential-operator/pull/580